### PR TITLE
correct osbapi service definition

### PR DESCRIPTION
**Pull request by MHBauer**
- Space before omitempty was semantically significant. Optional fields
   were being sent as "null"
 - Add documentation link to the v2.12 tag description of the Service.
 - Rearrange fields to match the documentation order.
 - remove extra newline

### CODE REVIEW

File: pkg/brokerapi/service_binding.go [hunk 1]
```diff
@@ -40,7 +40,6 @@ type BindingRequest struct {
 // CreateServiceBindingResponse represents a response to a service binding
 // request
 type CreateServiceBindingResponse struct {
-	// SyslogDrainURL string      `json:"syslog_drain_url, omitempty"`
```
Comment (duglin) [inline pkg/brokerapi/service_binding.go]: I think this was there on purpose as a reminder/placeholder

Comment (MHBauer) [reply pkg/brokerapi/service_binding.go]: doesn't look like it to me, it was the initial implementation.

Comment (MHBauer) [reply pkg/brokerapi/service_binding.go]: https://github.com/MHBauer/service-catalog/commit/5abaa81844d051891e9170a613b48b6078a50574#diff-c58f3f72bc2cfd04f51929bb4f19ce60R37

Comment (MHBauer) [reply pkg/brokerapi/service_binding.go]: Can add them all with omitempty if you want?
https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md#body-6

**Pull request merged**